pkg/api/api.go

```diff
@@ -0,0 +1,41 @@
+package api
+
+import (
+	"github.com/gorilla/mux"
+	"github.com/jeka2708/golang-training-enterprise/pkg/data"
+)
+
+type dataAPI struct {
+	data *data.DataEnterprise
+}
+
+func ServeUserResource(r *mux.Router, data data.DataEnterprise) {
+	api := &dataAPI{data: &data}
+	r.HandleFunc("/clients", api.getAllClients).Methods("GET")
+	r.HandleFunc("/divisions", api.getAllDivision).Methods("GET")
+	r.HandleFunc("/roles", api.getAllRole).Methods("GET")
+	r.HandleFunc("/services", api.getAllServices).Methods("GET")
+	r.HandleFunc("/works", api.getAllWork).Methods("GET")
+	r.HandleFunc("/works-clients", api.getAllWorkClient).Methods("GET")
+	r.HandleFunc("/workers", api.getAllWorkers).Methods("GET")
+
+	r.HandleFunc("/create-client", api.CreateUser).Methods("POST")
```

Comment:
incorrect name convention for paths. Read the following link: https://nordicapis.com/10-best-practices-for-naming-api-endpoints/

for example: instead of /create-client should be just /client. We can use different methods (get, post etc) for path with the same name